### PR TITLE
Simplify helloworld container with NC

### PR DESCRIPTION
**Pull request by luebken**
NC serves a simple html file.

Fixes: https://github.com/giantswarm/docs/issues/5

### CODE REVIEW

**Comment by luebken** [comment]
@marians could you beautify the html output?
@denderello RFR

**Comment by marians** [comment]
Cool! In general: Yes, I could. Due to other stuff on the table, Won't be able to do that in the short term (Monday). So please don't wait for me and merge.

**Comment by marians** [comment]
Just tested and got a blank page instead of the expected HTML page.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+MAINTAINER matthias@giantswarm.io
+
+ADD http-response http-response
+
+EXPOSE 80
+
+# A single command to serve a http response.
+# Inspired by http://www.commandlinefu.com/commands/view/9164/one-command-line-web-server-on-port-80-using-nc-netcat
+CMD while true; do nc -l -p 80 < http-response; done
```
Comment (denderello) [inline Dockerfile]: Why don't you use an `ENTRYPOINT` here?

Comment (denderello) [inline Dockerfile]: Also I don't think this will handle signals in any way because you have to set a `trap` in order to react on them.

**Comment by denderello** [comment]
Helps to make the image smaller but I would argue that `nc` is a real webserver that is capable to handle things like multiple connections. Well, it works. I don't have strong opinions and it solves the issue.

File: Dockerfile (hunk 1 above)
Comment (luebken) [inline Dockerfile]: Also @denderello pointed to https://github.com/denderello/jax-demo/blob/master/03-redis-sidekick/redis-heartbeat.service#L14

**Comment by luebken** [comment]
Closing this in favour of a Golang version: https://github.com/giantswarm/helloworld/pull/6

**Pull request closed without merging**